substrate/bin/node/runtime/src/genesis_config_presets.rs

```diff
@@ -0,0 +1,211 @@
+	let alice = Sr25519Keyring::Alice;
+	let bob = Sr25519Keyring::Bob;
+
+	// alice to ferdie
+	let endowed = Sr25519Keyring::well_known().map(|k| k.to_account_id()).collect::<Vec<_>>();
+
+	let patch = match id.as_ref() {
+		sp_genesis_builder::DEV_RUNTIME_PRESET => kitchen_sink_genesis(
+			vec![(
+				alice.to_account_id(),
+				alice.to_account_id(),
+				session_keys_from_seed(&alice.to_seed()),
+			)],
+			alice.to_account_id(),
+			endowed,
+			vec![validator(alice.to_account_id())],
+			None,
+		),
+		sp_genesis_builder::LOCAL_TESTNET_RUNTIME_PRESET => kitchen_sink_genesis(
+			vec![
+				(
+					alice.to_account_id(),
+					alice.to_account_id(),
+					session_keys_from_seed(&alice.to_seed()),
+				),
+				(bob.to_account_id(), bob.to_account_id(), session_keys_from_seed(&bob.to_seed())),
+			],
+			alice.to_account_id(),
+			endowed.clone(),
+			vec![validator(alice.to_account_id()), validator(bob.to_account_id())],
+			None,
+		),
+		_ => return None,
+	};
+
+	Some(
+		serde_json::to_string(&patch)
+			.expect("serialization to json is expected to work. qed.")
+			.into_bytes(),
+	)
+}
+
+/// List of supported presets.
+pub fn preset_names() -> Vec<PresetId> {
+	vec![
+		PresetId::from(sp_genesis_builder::DEV_RUNTIME_PRESET),
+		PresetId::from(sp_genesis_builder::LOCAL_TESTNET_RUNTIME_PRESET),
+	]
+}
+
+/// Sets up the `account` to be a staker of validator variant as supplied to the
+/// staking config.
+pub fn validator(account: AccountId) -> Staker {
+	// validator, controller, stash, staker status
+	(account.clone(), account, STASH, StakerStatus::Validator)
+}
+
+/// Extract some accounts from endowed to be put into the collective.
+fn collective(endowed: &[AccountId]) -> Vec<AccountId> {
+	const MAX_COLLECTIVE_SIZE: usize = 50;
+	let endowed_accounts_count = endowed.len();
+	endowed
+		.iter()
+		.take(((endowed_accounts_count + 1) / 2).min(MAX_COLLECTIVE_SIZE))
+		.cloned()
+		.collect()
+}
+
+pub fn session_keys(
+	grandpa: GrandpaId,
+	babe: BabeId,
+	im_online: ImOnlineId,
+	authority_discovery: AuthorityDiscoveryId,
+	mixnet: MixnetId,
+	beefy: BeefyId,
+) -> SessionKeys {
+	SessionKeys { grandpa, babe, im_online, authority_discovery, mixnet, beefy }
+}
+
+/// We have this method as there is no straight forward way to convert the
+/// account keyring into these ids.
+pub fn session_keys_from_seed(seed: &str) -> SessionKeys {
+	session_keys(
+		get_public_from_string_or_panic::<GrandpaId>(seed),
+		get_public_from_string_or_panic::<BabeId>(seed),
+		get_public_from_string_or_panic::<ImOnlineId>(seed),
+		get_public_from_string_or_panic::<AuthorityDiscoveryId>(seed),
+		get_public_from_string_or_panic::<MixnetId>(seed),
+		get_public_from_string_or_panic::<BeefyId>(seed),
+	)
+}
+
+fn get_public_from_string_or_panic<TPublic: Public>(s: &str) -> <TPublic::Pair as Pair>::Public {
+	TPublic::Pair::from_string(s, None)
+		.expect("Function expects valid argument; qed")
+		.public()
+}
```

Comment:
Could we re-use this function?
https://github.com/paritytech/polkadot-sdk/blob/c11b1f8526d704140f4329e68348c0b0965285de/substrate/primitives/core/src/crypto.rs#L425-L431

Reply:
Done f7769eb8434a47f4048133a097882744cff9be98